index.html

```diff
@@ -54,8 +53,8 @@ <h1 class="main-title">Find a gift for your wife or girlfriend</h1>
             </p>
             <button class="main-btn">FIND GIFT NOW</button>
           </div>
-          <div class="hero-col">
-            <img src="./images/happy_couple.png" alt="happy couple" />
+          <div class="hero-col happy-couple-image">
+            <img src="./images/happy_couple.webp" alt="happy couple" />
```

Comment:
???
Why are you using webp images without fallback? What about browsers support?